conans/client/conan_api.py

```diff
@@ -429,7 +429,18 @@ def export_pkg(self, path, name, channel, source_folder=None, build_folder=None,
         reference = ConanFileReference(name, version, user, channel)
         self._manager.export_pkg(reference, source_folder=source_folder, build_folder=build_folder,
                                  install_folder=install_folder, profile=profile, force=force)
+                                 
+    @api_method
+    def pack(self, reference, name, version, user, channel,
+             package_id, properties, pack_name, output_dir=".", target_scheme=None):
```

Comment:
Too many arguments. Same issue as above.

Reply:
Thanks a lot for the feedback. 

I'd appreciate if we do the review top-down, i.e. starting from the user experience, rather than the code implementation. 

Is the functionality exposed to the user appropriate?
Does it cover the most common use cases?
Should we add any other parameters or rename some?

I will fix the code as you propose in the next iteration, thanks for looking into it.

Reply:
I agree with this, lets focus on what it does first, code implementation second. Of course, style and other comments are good (specially the one about using the conan ``output``), but it would be very nice if you tried the functionality, did it work, the overall approach is good or not, just add a brief comment to the review: "everything looks good in this feature, just suggested a few implementation code changes". Many thanks to both! :)